pkg/option/config.go

```diff
@@ -2044,6 +2054,19 @@ func (c *DaemonConfig) K8sAPIDiscoveryEnabled() bool {
 	return c.k8sEnableAPIDiscovery
 }
 
+// K8sLeasesFallbackDiscoveryEnabled returns true if we should fallback to direct API
+// probing when checking for support of Leases in case Discovery API fails to discover
+// required groups.
+func (c *DaemonConfig) K8sLeasesFallbackDiscoveryEnabled() bool {
+	return c.k8sEnableAPIDiscovery
+}
+
+// EnableK8sLeasesFallbackDiscovery enables using direct API probing as a fallback to check
+// for the support of Leases when discovering API groups is not possible
```

Comment:
Nit: 

```suggestion
// for the support of Leases when discovering API groups is not possible.
```

(missing period at end)